types/onecolor/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for onecolor 3.1
+// Project: https://github.com/One-com/one-color
+// Definitions by: Azliya <https://github.com/Azliya>
+// Definitions: https://github.com/DefinitelyTyped/onecolor
+
+export = onecolor;
+declare function onecolor(clr: onecolor.ColorDefType): onecolor.Color;
+declare namespace onecolor {
```

Comment:
This can be used as global as well as module.. 
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.